Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -54,7 +54,8 @@ Tileset::Tileset(
                   _loadedTiles,
                   externals,
                   options.ellipsoid},
-              std::vector<CesiumAsync::IAssetAccessor::THeader>{},
+              std::vector<CesiumAsync::IAssetAccessor::THeader>(
+                  options.requestHeaders),
```

Comment:
`TilesetContentManager` is private, so let's change this constructor to be consistent with the others and not take the headers as a parameter.